views/networking/load_balancer/create.erb

```diff
@@ -0,0 +1,177 @@
+<% @page_title = "Create Load Balancer" %>
+
+<div class="space-y-1">
+  <%== render(
+    "components/breadcrumb",
+    locals: {
+      back: "#{@project_data[:path]}/networking?tab=load-balancers",
+      parts: [
+        %w[Projects /project],
+        [@project_data[:name], @project_data[:path]],
+        ["Load Balancers", "#{@project_data[:path]}/load-balancer"],
+        %w[Create #]
+      ]
+    }
+  ) %>
+  <%== render("components/page_header", locals: { title: "Create Load Balancer" }) %>
+</div>
+
+<div class="grid gap-6">
```

Comment:
Should we extend the inputs into the horizontal empty space?

<img width="1509" alt="Screenshot 2024-07-11 at 14 39 42" src="https://github.com/ubicloud/ubicloud/assets/993199/61416b63-ca1f-44db-9164-de8bd6a9a120">

Reply:
The problem with this view is that I don't want to put so many items on the same row. I think that kills the usability. If we keep it at 2 items and make the overall view where the items listed, longer, then we will have 200 char long text boxes for just 2 char integer. So, I don't have a solution to this. I think, most important thing is to make sure the text boxes on the same row looks side by side, for most screens. Is this good enough for you?
<img width="931" alt="Screenshot 2024-07-11 at 15 03 01" src="https://github.com/ubicloud/ubicloud/assets/6233557/efa91aff-8ee7-414c-8ab9-3a08537292c9">